### PR TITLE
Use APITimeout and RetryInterval constant in tests

### PR DESCRIPTION
**Pull request by e0ne**

### CODE REVIEW

**Comment by mlguerrero12** [comment]
LGTM

File: controllers/sriovnetworkpoolconfig_controller_test.go [hunk 1]
```diff
@@ -65,7 +66,7 @@ var _ = Describe("Operator", func() {
 					return err
 				}
 				return nil
-			}, timeout*3, interval).Should(Succeed())
+			}, util.APITimeout*3, util.RetryInterval).Should(Succeed())
```
Comment (adrianchiris) [inline controllers/sriovnetworkpoolconfig_controller_test.go:69]: this is odd that we need it. probably related to where this test is executed.

i mean we just spin up apiserver +etcd with envtest then a simple reconcile

**Review by adrianchiris — APPROVED** [review]

File: controllers/sriovnetworkpoolconfig_controller_test.go (hunk 1 above)
Comment (e0ne) [reply controllers/sriovnetworkpoolconfig_controller_test.go:69]: IMO, that's something we have to fix in the future in a sepatare PR

**Review by zeeke — APPROVED** [review]
I can't reproduce the error this PR is fixing on my local environment, but looking at the CI, it fixes the job.

LGTM

**Pull request merged**